pkg/standalone/standalone.go

```diff
@@ -307,6 +308,11 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		}
 		print.InfoStatusEvent(os.Stdout, "Use `%s ps` to check running containers.", runtimeCmd)
 	}
+	// TODO: remove below method when components-path flag is removed.
+	err = emptyAndCopyFiles(DefaultComponentsDirPath(), DefaultResourcesDirPath())
```

Comment:
Should also handle here when the components directory does not exist, since `emptyAndCopyFiles` will throw an error.

Reply:
aah.. thats why I had the logic this way in `emptyAndCopyFiles` - 

```
if _, err := os.Stat(componentsDirPath); err == nil
```
otherwise I was returning nil..

Reply:
Yes, but we should error out for anything other than `os.IsNotExist`.